libevm/EVMC.h

```diff
@@ -70,14 +70,13 @@ class EVM
     Result execute(ExtVMFace& _ext, int64_t gas)
     {
         auto mode = toRevision(_ext.evmSchedule());
+        evm_call_kind kind = _ext.isCreate ? EVM_CREATE : EVM_CALL;
         uint32_t flags = _ext.staticCall ? EVM_STATIC : 0;
```

Comment:
Should we make any assumptions for evmc here that `isCreate` and `staticCall` is mutually exclusive?

Reply:
I'd rather not. If we want that, the good way of doing it, would be to extend the "call kind" with some additional kinds...

Reply:
Either new kinds or an assert here.